workflow-init/profile/hawk.yml

```diff
@@ -219,6 +219,15 @@ component:
     kustomize:
       path: plattform/processing/case-sink
       overlays: *defaultOverlays
+    modules:
+      model:
+        parameters:
+          <<: *defaultParameters
+          kustomize: github
+      app:
```

Comment:
If we want to build the two modules separately we need a dedicated `java-version: 17` for the app. The model is a Java lib that we do not use at the moment.

Reply:
The new base build for this repo currently uses the module name to generate the test report, we don't use java-version property yet. I will add it later when it can be used